File: sdks/java/src/main/java/com/puxbay/models/ReturnRequest.java
package com.puxbay.models;

import com.google.gson.annotations.SerializedName;

public class ReturnRequest {
    private String id;
    private String order;
    private String status;
    private String reason;
    
    @SerializedName("refund_amount")
    private double refundAmount;
    
    @SerializedName("created_at")
    private String createdAt;
    
    @SerializedName("updated_at")
    private String updatedAt;

    public String getId() { return id; }
    public void setId(String id) { this.id = id; }
    
    public String getOrder() { return order; }
    public void setOrder(String order) { this.order = order; }
    
    public String getStatus() { return status; }
    public void setStatus(String status) { this.status = status; }
    
    public String getReason() { return reason; }
    public void setReason(String reason) { this.reason = reason; }
    
    public double getRefundAmount() { return refundAmount; }
    public void setRefundAmount(double refundAmount) { this.refundAmount = refundAmount; }
    
    public String getCreatedAt() { return createdAt; }
    public void setCreatedAt(String createdAt) { this.createdAt = createdAt; }
    
    public String getUpdatedAt() { return updatedAt; }
    public void setUpdatedAt(String updatedAt) { this.updatedAt = updatedAt; }
}

File: sdks/kotlin/src/main/kotlin/com/puxbay/models/ReturnRequest.kt
package com.puxbay.models

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class ReturnRequest(
    val id: String,
    val order: String,
    val status: String,
    val reason: String,
    @SerialName("refund_amount") val refundAmount: Double,
    @SerialName("created_at") val createdAt: String,
    @SerialName("updated_at") val updatedAt: String
)
